src/Persistence.LinnApps/ServiceDbContext.cs

```diff
@@ -1515,6 +1515,12 @@ private void BuildRequisitionHeaders(ModelBuilder builder)
             e.Property(r => r.ToLocationId).HasColumnName("TO_LOCATION_ID");
             e.Property(r => r.Document1Name).HasColumnName("DOC1_NAME");
             e.Property(r => r.PartNumber).HasColumnName("PART_NUMBER").HasMaxLength(14);
+            e.Property(r => r.Document1Name).HasColumnName("DOC1_NAME");
```

Comment:
this propery is alread mapped two lines above

Reply:
Small typo, just fixing in next commit - meant to be the line.. Sorting dates too.